lua/lazyvim/plugins/extras/test/core.lua

```diff
@@ -0,0 +1,51 @@
+return {
+  {
+    "nvim-neotest/neotest",
+    dependencies = {
+      -- which key integration
+      {
+        "folke/which-key.nvim",
+        opts = {
+          defaults = {
+            ["<leader>t"] = { name = "+test" },
+          },
+        },
+      },
+    },
+    config = function(_, opts)
+      -- get neotest namespace (api call creates or returns namespace)
+      local neotest_ns = vim.api.nvim_create_namespace("neotest")
+      vim.diagnostic.config({
+        virtual_text = {
+          format = function(diagnostic)
+            local message = diagnostic.message:gsub("\n", " "):gsub("\t", " "):gsub("%s+", " "):gsub("^%s+", "")
+            return message
+          end,
+        },
+      }, neotest_ns)
+      local group = vim.api.nvim_create_augroup("lazyvim_neotest_close_with_q", { clear = true })
```

Comment:
no need to create a separate autocmd here. It's ok to add `neotest-output` to the default close_with_q autocmd